content/english/_index.md

```diff
@@ -4,4 +4,6 @@ title: Home
 
 The Swedish COVID-19 Data Portal provides information, guidelines, tools and services to support researchers in utilising Swedish and European infrastructures for data sharing, in particular the [European COVID-19 Data Portal](https://covid19dataportal.org).
 
-> För information på svenska, [klicka här](/sv/)
+There are currently [5 open positions](/about/#open-positions) at this Data Portal. Closing date is July 31 2020.
```

Comment:
```suggestion
There are currently [5 open job applications](/about/#open-positions) to work on this data portal. Closing date is July 31 2020.
```

Reply:
Could also wrap it in a blue box to make it stand out a little:

```suggestion
<div class="alert alert-info">
    There are currently [5 open job applications](/about/#open-positions) to work on this data portal. Closing date is July 31 2020.
</div>
```

Reply:
Or get _really_ fancy and have a box and an [icon](https://fontawesome.com/icons/users-cog?style=solid)
```suggestion
<div class="alert alert-info">
    <i class="fas fa-users-cog"></i> There are currently [5 open job applications](/about/#open-positions) to work on this data portal. Closing date is July 31 2020.
</div>
```